templates/registration/two_factor_auth.html

```diff
@@ -81,8 +81,8 @@
             {% endif %}
 
             {% if request.profile.is_totp_enabled %}
-                <div><label class="inline-header grayed">{{ _('Enter the 6-digit code generated by your app:') }}</label></div>
-                <div id="totp-token-container"><span class="fullwidth">{{ form.totp_token }}</span></div>
+	            <div><label class="inline-header grayed">{{ _('Enter the 6-digit code generated by your app <br> or one of your 16-digit scratch codes:') }}</label></div>
```

Comment:
```suggestion
	       <div><label class="inline-header grayed">{{ _('Enter the 6-digit code generated by your app <br> or one of your 16-character scratch codes:') }}</label></div>
```

You will need to double check on the indenting, as indenting a suggestion is hard.